src/app/components/pagination-component/pagination-component.sass

```
@@ -6,16 +6,18 @@
   display: grid
   justify-content: center
 
-ngb-pagination ::ng-deep .page-item .page-link
-  border: none !important
-  display: flex
-  justify-content: center
-  align-items: center
-  border-radius: 6px
+ngb-pagination
+  &::ng-deep .page-item .page-link
+    border: none !important
+    display: flex
+    justify-content: center
+    align-items: center
+    border-radius: 6px
   & span
     display: none
```

Could you please remove parent `&` here? It's not needed

```suggestion
  span
    display: none
```